### DISCUSSION TITLE
Email without credentials

### DISCUSSION THREAD

**Discussion by tmeltdowner** [EvotecIT/PSWriteHTML#268] [Q&A] [answered]
Our STMP Server authenticates based on the incoming IP address, not a username password.  When we try to use Email function without credentials, the email is never sent.  If we try to use Credentials, it still fails to send.

Does the Email function allow an email to be sent without credentials?

**Comment by PrzemyslawKlys** [comment, marked as answer]
It should do that. Make sure to use the proper port (25?). By default it uses 587.

**Comment by tmeltdowner** [comment]
Thank you, I had to specify port 25!  That was it!

**Reply by tmeltdowner** [reply]
I can't click Mark As Answer, a pop-up on this website wont go away...
![image](https://user-images.githubusercontent.com/34780310/130136873-a6e76643-10e3-4815-baff-febbe69213d5.png)

**Reply by PrzemyslawKlys** [reply]
It's weird sometimes. In other dicussions i can't mark answers at all ;)
